lib/service_manager.dart

```diff
@@ -370,6 +372,20 @@ class ServiceExtensionManager {
     }
   }
 
+  dynamic _getExtensionValueFromJson(String name, String valueFromJson) {
+    final expectedValueType =
+        extensions.toggleableExtensionsWhitelist[name].enabledValue.runtimeType;
+    switch (expectedValueType) {
+      case bool:
+        return valueFromJson == 'true' ? true : false;
+      case int:
```

Comment:
nit: shouldn't int use int.parse for the int case?
That won't matter on the web but will blow up if we run this in flutter at some point.

Reply:
technically I think we can drop the int case because the numeric service extension is only set with a double.

Reply:
hmm actually no we do need it. I'll add int.parse.